lib/crowbar/client/config.rb

```diff
@@ -108,18 +109,35 @@ def default_upgrade_versions
         return "7-to-8" if File.exist?("/var/lib/crowbar/upgrade/7-to-8-upgrade-running")
 
         # if upgrade has not been started, check the system version
-        os_release_file = "/etc/os-release"
+        os_release = Crowbar::Client::Util::OsRelease.fields
 
-        if File.exist?(os_release_file)
-          os_release = Hash[
-            File.open(os_release_file).read.scan(/(\S+)\s*=\s*"([^"]+)/)
-          ]
-          return "6-to-7" if os_release["VERSION_ID"] == "12.1"
-        end
+        return "6-to-7" if os_release["VERSION_ID"] == "12.1"
```

Comment:
@jsuchome with the cloud_config piece now available we could replace this with something like `return "6-to-7" if default_cloud_version == "6"`. The meaning is not 100% the same in case of overrides but I'm not sure if this matters.

Reply:
I agree with changing to `if default_cloud_version == "6"` Overrides for upgrade are different ENVs, right? So it should not matter...

Do you think adding a check for 8-9 branch belongs to separate PR?

Reply:
The difference is that without the `== "6"` check calling `CROWBAR_CLOUD_VERSION=6 crowbarctl upgrade` would still show the database commands (when run on e.g. cloud7 node). Only `CROWBAR_UPGRADE_VERSIONS=6-to-7 crowbarctl upgrade` would hide them. I think it makes sense to "pass" this override from `CROWBAR_CLOUD_VERSION` to `CROWBAR_UPGRADE_VERSIONS`. Will add it.